frontend/test/locales/ar.json

```
@@ -556,7 +556,8 @@
         "html": "لغة البرمجة",
         "plain": "نص عادي",
         "copyText": "نسخ النص",
-        "copied": "نسخ!"
+        "copied": "نسخ!",
+        "xml": "XML"
```

Let's use the Arabic transliteration of "xml" here. Also noting, for a separate issue, that the translation of "html" is quite incorrect above.

```suggestion
        "xml": "إك سم إل"
```

---

This has been updated.